src/radio/src/Radio.js

```diff
@@ -119,18 +29,24 @@ class Radio extends PureComponent {
       size,
       isRequired,
       appearance,
-      ...props
-    } = this.props
+      ...rest
+    } = props
+    const theme = useTheme()
     const themedClassName = theme.getRadioClassName(appearance)
 
+    const handleChange = useCallback(
+      event => onChange(event, event.target.checked),
```

Comment:
Curious if we should stop shimming the event handler?

Why don't we let consumers pass in a normal `onChange` handler?

Reply:
Unsure -- I was surprised to see this API different than the one on `<Checkbox />` for example. Since this is going to v5, I'm fine just removing this and letting this and `<Checkbox />` have the same behavior.

Reply:
👍 would be happy with normal event handlers everywhere, for consistency

Reply:
+1